numpy/ma/core.py

```diff
@@ -4343,7 +4344,7 @@ def imag(self):
         Return the imaginary part of the masked array.
 
         The returned array is a view on the imaginary part of the `MaskedArray`
-        whose `get_imag` method is called.
+        whose `imag` method is called.
```

Comment:
now not a method but a `property`

Reply:
So change to say "whose `imag` property is called"? Or something else?

Reply:
How about just
```diff
- of the `MaskedArray` whose `get_imag` method is called.
+ of this `MaskedArray`.
```